docker/1.13/Dockerfile.gpu

```diff
@@ -1,70 +1,138 @@
-ARG TFS_VERSION
-
-FROM tensorflow/serving:${TFS_VERSION}-gpu as tfs
 FROM nvidia/cuda:10.0-base-ubuntu16.04
+
+LABEL maintainer="Amazon AI"
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
 
-COPY --from=tfs /usr/bin/tensorflow_model_server /usr/bin/tensorflow_model_server
+# Add arguments to achieve the version, python and url
+#  PYTHON=python for 2.7
+#  PYTHON=python3 for 3.5, 3.6 is not available directly on 16.04
```

Comment:
Can you combine the python and pip comments into just one line?

Reply:
When I do:

`ARG PYTHON=python3 PIP=pip3`, I got the following error:

building image...
Sending build context to Docker daemon  63.02kB
Error response from daemon: Dockerfile parse error line 12: ARG requires exactly one argument